pkg/resourceinterpreter/defaultinterpreter/aggregatestatus.go

```diff
@@ -292,9 +295,7 @@ func aggregatePodStatus(object *unstructured.Unstructured, aggregatedStatusItems
 			return nil, err
 		}
 
-		if temp.Phase != corev1.PodRunning {
-			runningFlag = false
-		}
+		podPhases.Insert(string(temp.Phase))
```

Comment:
Just a question, if it possible that `temp.Phase == ""`?

Reply:
I don't know.
If it is `nil`, it means that there is one more state of `PodPhase`

Reply:
@XiShanYongYe-Chang Please help to check it.

Reply:
In the [getPhase](https://github.com/kubernetes/kubernetes/blob/d14ba948ef63769e9767aebd5a08171832a1bbf6/pkg/kubelet/kubelet_pods.go#L1328) function, the default phase is pending:

https://github.com/kubernetes/kubernetes/blob/d14ba948ef63769e9767aebd5a08171832a1bbf6/pkg/kubelet/kubelet_pods.go#L1423-L1425

Reply:
Got it.

Reply:
Thanks!  @XiShanYongYe-Chang